.github/workflows/continuous-integration-test.yml

```diff
@@ -0,0 +1,76 @@
+name: build
+on: [push, pull_request, workflow_dispatch]
+
+env:
+  TERM: xterm-256color
```

Comment:
Why do you need a `TERM` variable?

Reply:
Because I use `tput` to display messages in the bash scripts